templates/validator/overview.html

```diff
@@ -235,12 +235,14 @@
       <span id="depositCount" style="cursor: pointer;" data-toggle="tooltip" title="Deposits made to the deposit contract"><i class="fas fa-wallet"></i> {{ .DepositsCount }}</span>
     </div>
     <div class="mx-3">
-      <span id="withdrawalCount" style="cursor: pointer;" data-toggle="tooltip" {{ if .IsWithdrawableAddress }}title="Withdrawals processed for this validator"{{ else }}title="To receive payouts you need 0x01 withdrawal credentials"{{ end }}>
+      <span id="withdrawalCount" style="cursor: pointer;" data-toggle="tooltip" {{ if or (.IsWithdrawableAddress) (not .CappellaHasHappened) }}title="Withdrawals processed for this validator"{{ else }}title="To receive payouts you need 0x01 withdrawal credentials"{{ end }}>
```

Comment:
I think we should hide the whole ` <div class="mx-3"> ` block before cappella.
Does not make sense to just show the icon. 

Then you can remove the two CappellaHasHappened conditions within the block

![image](https://user-images.githubusercontent.com/125363940/221533901-9a292eb6-81c8-4b2c-8378-b2ca46524364.png)

Reply:
If we hide the icon, then we should hide the withdrawal tab too.

Reply:
Good suggestions, done